pkgs/development/python-modules/labgrid/default.nix

```diff
@@ -6,13 +6,15 @@
 , jinja2
 , lib
 , mock
+, openssh
 , packaging
 , pexpect
 , psutil
-, pyserial
+, pyserial-labgrid
```

Comment:
`pyserial-labgrid` is not required for nixpkgs, since the patches are included within pyserial, see https://github.com/NixOS/nixpkgs/pull/113388.

Reply:
Oh, nice. I'll put `pyserial-labgrid` in `pythonRemoveDeps` then.

Reply:
Done.